desktop/e2e/run.js

```diff
@@ -18,6 +18,8 @@ const APP_ARGS = [
 	'--disable-http-cache',
 	'--start-maximized',
 	'--remote-debugging-port=9222',
+	'--disable-setuid-sandbox',
```

Comment:
Without this Chrome throws lots of problems. This was also done in CircleCI at the image config in https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/node/images/12.18.4-stretch/browsers/Dockerfile#L46